src/api/src/graphql/resolvers/SerialMonitor.resolver.ts

```diff
@@ -35,7 +35,10 @@ export default class SerialMonitorResolver {
     input: SerialConnectionConfigInput
   ): Promise<SerialPortConnectResult> {
     try {
-      await this.serialMonitorService.connect(input.port, input.baudRate);
```

Comment:
Let's fix the type of baud rate upstream, no need to convert it to number over here.